apps/chat/src/store/models/models.epics.ts

```diff
@@ -317,8 +317,8 @@ const addInstalledModelsEpic: AppEpic = (action$, state$) =>
                   of(
                     UIActions.showSuccessToast(
                       payload.references.length > 1
-                        ? translate('Models added to my workspace')
-                        : translate('The model added to my workspace'),
+                        ? translate('The agents added to my workspace')
+                        : translate('The agent added to my workspace'),
```

Comment:
translate(`The agent${payload.references.length > 1 ? 's' : ''} added to my workspace`)